p4-16/psa/PSA.mdk

```diff
@@ -424,6 +429,15 @@ translation performed may lead to functional errors in your program:
 
 + Do a table key match on a subset of the bits, or a range match.
 + Compare port values with relational operators like `<` or `>`.
++ Compare port values to specific numeric literal values like 0 or
+  0xff.  It is recommended instead to compare their values by using
+  them as table key fields, or parser value set key fields, against
+  values installed by the control plane (which have been translated to
+  the corresponding device-specific value as determined by the
+  device's agent software).  It is also reasonable to compare port
+  values for equality against the symbolic constant values
+  `PSA_PORT_CPU` or `PSA_PORT_RECIRCULATE`, which have target-specific
+  numeric values.
```

Comment:
This addition isn't really a new restriction -- mainly calling out one that could have been described in PSA v1.0 but was not.  It generated some discussion in a P4-API working group, so I decided it would be best to explicitly describe it with the other recommended restrictions on use of such translated values in P4 programs.